### PR TITLE
PP-626/RelayClient-RelayPricer

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

- refactor RelayPricer with ethers if necessary

## Why

- Project refactoring

## Refs

- [PP-626](https://rsklabs.atlassian.net/browse/PP-626)

- (optional: include links to other issues, PRs, tickets, etc)

### CODE REVIEW

File: test/RelayPricer.test.ts [hunk 1]
```diff
@@ -1,180 +1,204 @@
...
+import chaiAsPromised from 'chai-as-promised';
+import { BigNumber as BigNumberJs } from 'bignumber.js';
+
+import RelayPricer from '../src/RelayPricer';
+import type BaseExchangeApi from '../src/api/pricer/ExchangeApi';
+
+chai.use(chaiAsPromised);
+
+const randomBigNumberJs = (base: number) =>
+    BigNumberJs((Math.random() * base));
```
Comment (antomor) [inline test/RelayPricer.test.ts:12]: NIT: I'd change the argument name to be `max` and furthermore, I don't think we need the double parentheses.
```suggestion
const randomBigNumberJs = (max: number) =>
    BigNumberJs(Math.random() * max);
```

**Review by antomor — APPROVED** [review]
@franciscotobar thank you for the changes, i've just left a minor comment.

**Review by ironFe93 — APPROVED** [review]
lgtm

File: test/RelayPricer.test.ts (hunk 1 above)
Comment (franciscotobar) [reply test/RelayPricer.test.ts]: updated

**Pull request merged**